Week-13/weatherApi/service.js

```diff
@@ -0,0 +1,74 @@
+import {BASE_URL, WEATHER_API} from "./constants/endpoint.js";
+import {PAGINATION_COUNT} from "./constants/global.js";
+import fetch from "node-fetch";
+import _lodash from "lodash";
+const {chunk} = _lodash;
+
+export const getMultipleCities = async ({res, cities}) => {
+  try {
+    let weatherResponse = [];
+    for (const element of cities) {
+      let cityWeatherResponse = await fetch(
+        `${BASE_URL}${WEATHER_API.CURRENT}?key=${process.env.WEATHER_API_KEY}&q=${element}&aqi=no`
+      );
+      let cityWeather = await cityWeatherResponse.json();
+      if (cityWeather.error) {
+        throw new Error(cityWeather.error.message);
+      } else {
+        weatherResponse = [...weatherResponse, cityWeather];
+      }
+    }
+    const resp = JSON.stringify({
```

Comment:
fetching the data from a particular ui, getting the response json, checking for error and returning the response seems to be the common part in all the methods. Probably we can abstract this part out and use a common method in all these methods. Also, it is a better practice to use a service as a class with all these static methods. The base api url could be a private member of that class. This would lead to a better code if you had to use multiple services like weather api, youtube api, firebase api. Then you can just import the service class and use the methods. 
Also, whenever you get time, try using express instead of vanilla node js. Express makes the code readability much better and will also provide you a lot of different middle wares.